client/app/dbaas/metrics/dashboard/chart-pie/metrics-chart-pie.component.js

```diff
@@ -8,6 +8,11 @@
             this.ringSize = 2 * Math.PI * this.rayon;
         }
 
+        $onInit () {
+            if (isNaN(this.text)) {
```

Comment:
this.text = this.text || "0"

Reply:
This has already been tested and won't work. Because there's something returned by this.text which could be something else than a number.

Reply:
if someone makes the mistakes of passing the wrong value (where this.text is not a number) he'll struggle to understand what is going on.  This is why I wouldn't replace the value unless it is not provided in the first place.